pkg/mco/kubeletconfig.go

```diff
@@ -0,0 +1,268 @@
+package mco
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	mcv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubeletconfigv1beta1 "k8s.io/kubelet/config/v1beta1"
+)
+
+// Builder provides struct for KubeletConfig Object which contains connection to cluster
+// and KubeletConfig definitions.
+type Builder struct {
```

Comment:
Since this is in the mco package with other structs. we should call it something like KubeletConfigBuilder

Reply:
it was, you did not react and I changed it half an our ago